test/functional/setup.js

```diff
@@ -91,9 +91,10 @@ before(function () {
         })
         .then(function () {
             global.runTests = function (fixture, testName, opts) {
-                var report = '';
+                var report       = '';
                 var runner = testCafe.createRunner();
                 var fixturePath = path.join(path.dirname(caller()), fixture);
+                var skipJsErrors = opts && opts.skipJsErrors;

```

Comment:
Incorrect var declaration formatting. Please, setup WS formatting according to our rules